site/docs-md/basics/configuring-your-app.md

```diff
@@ -41,7 +41,14 @@ The current ones you might configure are:
     // If you don't configure it, a random port will be assigned and persisted.
     "port": "8787",
     // You can make the app to load an external url (i.e. to live reload)
-    "url": "http://192.168.1.33:8100"
+    "url": "http://192.168.1.33:8100",
+    // Normally all external URLs are opened in the browser. By setting this option, you tell
+    // Capacitor to open URLs belonging to these hosts inside its WebView.
+    "allowedHosts": [
```

Comment:
I wonder if `allowedHosts` is the right name for this config. My first impression of this is that it meant allowed URLs I can load in general, including fetch, etc.

Maybe `loadHostsInWebView`?

Reply:
@mlynch ok, I've made the change that you've suggested, and also rebased the PR to the current master.